cmd/dashboard/main.go

```diff
@@ -102,7 +129,14 @@ func main() {
 	controllers.StartKubeControllers(resource.K8sClient, resyncDur, dashboardConfig.installNamespace, routerHandler, ctx.Done())
 
 	logging.Log.Infof("Creating server and entering wait loop")
-	server := &http.Server{Addr: dashboardConfig.port, Handler: routerHandler}
+	CSRF := csrf.Protect(
+		getCSRFAuthKey(),
```

Comment:
This part's interesting, so while the return value doesn't get checked here, IDK why getCSRFAuthKey would ever error so this *should* never be nil 🤔

Reply:
if for some reason the system's random number generator is unable to provide sufficient output it'll fail and we return nil, should be very unlikely to happen though and gorilla/csrf has some code to attempt to recover from a missing key anyway.